src/logic/operate.js

```diff
@@ -1,25 +1,35 @@
-/* eslint-disable import/extensions */
+/* eslint-disable prefer-destructuring */
 import Big from 'big.js';
 
-function operate({ numberOne, numberTwo, operation }) {
+function operate(numberOne, numberTwo, operation) {
   let total;
   const opertionCalc = ['+', '-', 'x', '÷', '%'];
+  let x = 0;
+  let y = 0;
+
+  if (numberOne != null) {
+    x = Big(numberOne);
+  }
+
+  if (numberTwo != null) {
+    y = Big(numberTwo);
+  }
 
   switch (operation) {
     case opertionCalc[0]:
-      total = Big(numberOne).plus(numberTwo);
+      total = x.plus(y);
       break;
     case opertionCalc[1]:
-      total = Big(numberOne).minus(numberTwo);
+      total = y.minus(x);
       break;
     case opertionCalc[2]:
-      total = Big(numberOne).times(numberTwo);
+      total = x.times(y);
       break;
     case opertionCalc[3]:
-      total = Big(numberOne).div(numberTwo);
+      total = y.div(x);
       break;
```

Comment:
Manage division by ZERO. You can display an error on screen or something like `NaN` or `INFINITY`

![Screen Shot 2020-05-02 at 4 33 00 PM](https://user-images.githubusercontent.com/40712459/80868391-24e01300-8c92-11ea-8573-10e4bd689cf9.png)